core/chains/evm/client/node.go

```diff
@@ -61,6 +61,10 @@ var (
 		Name: "evm_pool_rpc_node_calls_success",
 		Help: "The approximate total number of successful RPC calls for the given RPC node",
 	}, []string{"evmChainID", "nodeName"})
+	promEVMPoolSendTransactionTiming = promauto.NewHistogramVec(prometheus.HistogramOpts{
```

Comment:
What histogram buckets does this default to?

Reply:
Good thing you mention this because the defaults look completely off:

```
evm_pool_rpc_node_send_tx_execution_time_bucket{evmChainID="80001",isSendOnly="false",nodeName="primary-0-80001",rpcHost="xxx",le="0.005"} 0
evm_pool_rpc_node_send_tx_execution_time_bucket{evmChainID="80001",isSendOnly="false",nodeName="primary-0-80001",rpcHost="xxx",le="0.01"} 0
evm_pool_rpc_node_send_tx_execution_time_bucket{evmChainID="80001",isSendOnly="false",nodeName="primary-0-80001",rpcHost="xxx",le="0.025"} 0
evm_pool_rpc_node_send_tx_execution_time_bucket{evmChainID="80001",isSendOnly="false",nodeName="primary-0-80001",rpcHost="xxx",le="0.05"} 0
evm_pool_rpc_node_send_tx_execution_time_bucket{evmChainID="80001",isSendOnly="false",nodeName="primary-0-80001",rpcHost="xxx",le="0.1"} 0
evm_pool_rpc_node_send_tx_execution_time_bucket{evmChainID="80001",isSendOnly="false",nodeName="primary-0-80001",rpcHost="xxx",le="0.25"} 0
evm_pool_rpc_node_send_tx_execution_time_bucket{evmChainID="80001",isSendOnly="false",nodeName="primary-0-80001",rpcHost="xxx",le="0.5"} 0
evm_pool_rpc_node_send_tx_execution_time_bucket{evmChainID="80001",isSendOnly="false",nodeName="primary-0-80001",rpcHost="xxx",le="1"} 0
evm_pool_rpc_node_send_tx_execution_time_bucket{evmChainID="80001",isSendOnly="false",nodeName="primary-0-80001",rpcHost="xxx",le="2.5"} 0
evm_pool_rpc_node_send_tx_execution_time_bucket{evmChainID="80001",isSendOnly="false",nodeName="primary-0-80001",rpcHost="xxx",le="5"} 0
evm_pool_rpc_node_send_tx_execution_time_bucket{evmChainID="80001",isSendOnly="false",nodeName="primary-0-80001",rpcHost="xxx",le="10"} 0
evm_pool_rpc_node_send_tx_execution_time_bucket{evmChainID="80001",isSendOnly="false",nodeName="primary-0-80001",rpcHost="xxx",le="+Inf"} 1
evm_pool_rpc_node_send_tx_execution_time_sum{evmChainID="80001",isSendOnly="false",nodeName="primary-0-80001",rpcHost="xxx"} 1.84470958e+08
evm_pool_rpc_node_send_tx_execution_time_count{evmChainID="80001",isSendOnly="false",nodeName="primary-0-80001",rpcHost="xxx"} 1
```

Reply:
I notice that our other `*_time` metrics are gauges, but I'm not a big fan of that because then we can't really see things like percentiles (e.g what is the p50 time vs. p95 time, etc.) and we also can't see counts (e.g how many times was SendTransaction called).

Reply:
Fixed the time buckets, the defaults were wrong. Opted for 50ms to 8s increasing by 2x everytime.